tool.gpt

```diff
@@ -61,7 +46,7 @@ open http://localhost:${PORT}
 
 ---
 
-name: start-file
+name: open-file-windows
```

Comment:
renamed to make it more obvious that these toosl are just the "windows variants" of the others.

Reply:
Eh, OK.

Reply:
Well, I can switch it back if we're feeling "meh" about it. Lmk